dz9_ex2.py

```diff
@@ -0,0 +1,16 @@
+# інтератор числа Фібоначчі
```

Comment:
It is not an iterator but using of the `for` loop to get `n` element in fibo sequence

Reply:
add file dz9_ex2.py